regtests/run_spark_sql.sh

```diff
@@ -16,11 +16,19 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+
+# -----------------------------------------------------------------------------
+# Purpose: Launch the Spark SQL shell to interact with Polaris.
+# -----------------------------------------------------------------------------
 #
+# Usage:
+#   Without arguments: Runs against a catalog backed by the local filesystem
```

Comment:
I still think this should be a more traditional usage string
```shell
./run_spark_sql.sh [S3-location AWS-IAM-role]
```

Reply:
Fixed